flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/JoinTestPrograms.java

```diff
@@ -107,7 +107,7 @@ public class JoinTestPrograms {
 
     static {
         NON_WINDOW_INNER_JOIN =
-                TableTestProgram.of("non-window-inner-join", "test non-window inner join")
+                TableTestProgram.of("join-non-window-inner-join", "test non-window inner join")
```

Comment:
Hope you won't find it nitpicking, but can we leave just a single occurence of `join`? I think `join-non-window-inner` still makes sense, but I am not a native speaker. If you find the duplication better leave it as is.

Reply:
Yeah, I don't like the duplication either...  Both options are unfortunate; I find this one slightly better.  Can we keep it as-is?

Reply:
👍